addons/captives/XEH_postInit.sqf

```diff
@@ -35,6 +36,22 @@ if (isServer) then {
 
 if (!hasInterface) exitWith {};
 
+    //Add Keybind:
```

Comment:
Many spaces. This comment is kind of superfluous anyways, it's seen immediately it's a keybind thing.